js/index.js

```diff
@@ -22,4 +22,9 @@ $( document ).ready(function() {
       return false;
     })
 
+  $('body').on('click', '.email-master', function(){
+      $(this).css("background-color", "#ddd");
```

Comment:
It's typically better to add a class that has the style of `background-color: #ddd;`, rather than adding the css here. I'd make a class called `.active` and add it to the clicked element.